backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/sort/impl/GeoPointFieldSortBuilder.java

```diff
@@ -4,19 +4,22 @@
  * License: GNU Lesser General Public License (LGPL), version 2.1 or later
  * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
  */
-package org.hibernate.search.backend.lucene.search.sort.impl;
+package org.hibernate.search.backend.lucene.types.sort.impl;
 
 import java.lang.invoke.MethodHandles;
 
+import org.apache.lucene.document.LatLonDocValuesField;
 import org.hibernate.search.backend.lucene.logging.impl.Log;
-import org.hibernate.search.backend.lucene.types.sort.impl.LuceneFieldSortContributor;
+import org.hibernate.search.backend.lucene.search.sort.impl.AbstractSearchSortBuilder;
+import org.hibernate.search.backend.lucene.search.sort.impl.LuceneSearchSortBuilder;
+import org.hibernate.search.backend.lucene.search.sort.impl.LuceneSearchSortCollector;
 import org.hibernate.search.engine.logging.spi.EventContexts;
 import org.hibernate.search.engine.search.dsl.sort.SortOrder;
 import org.hibernate.search.engine.search.sort.spi.DistanceSortBuilder;
 import org.hibernate.search.engine.spatial.GeoPoint;
 import org.hibernate.search.util.impl.common.LoggerFactory;
 
-class DistanceSortBuilderImpl extends AbstractSearchSortBuilder
+public class GeoPointFieldSortBuilder extends AbstractSearchSortBuilder
```

Comment:
Not sure this new name is a great one; first because this class implements `DistanceSortBuilder`, not `FieldSortBuilder`, second because it gives the false impression that this will sort by the GeoPoint's value somehow, and third because it's inconsistent with how the distance projection builders are named.

Perhaps what you're after is making the various `GeoPoint`-related classes sort nicely in the package, in which case I would suggest `GeoPointDistanceSortBuilder`?